.github/workflows/build_ubuntu-22.04.sh

```diff
@@ -52,5 +52,5 @@ export INSTALL_PREFIX=$1
     --with-fftw \
     --with-netcdf
 
-eval $makecmd
+"${makecmd[@]}"
```

Comment:
```suggestion
eval "$makecmd"
```

Reply:
This isn't quite the same. When having the quoted $makecmd, the arguments aren't split. Having unquoted $makecmd is problematic for globs, or all reasons why you'd want to quote variables.

Double quoted array with `[@]`, is equivalent of double quoting each element. That would allow to properly handle a path with spaces as a single argument. (https://www.shellcheck.net/wiki/SC2068)

See https://www.shellcheck.net/wiki/SC2086 for general quoting tips (near the end), and https://www.shellcheck.net/wiki/SC2068 and https://www.shellcheck.net/wiki/SC2048 and 
https://www.shellcheck.net/wiki/SC2294

Reply:
This goes with above suggestion.